docs/api/prompts.md

```diff
@@ -1 +1 @@
-::: outlines.prompts
+::: outlines.templates
```

Comment:
Forgot to rename the file too

Reply:
They appear as renamed here?

Reply:
File name*
<img width="295" alt="Screenshot 2025-02-24 at 09 39 42" src="https://github.com/user-attachments/assets/01a73678-fc25-411c-b76e-64e39f87c526" />

Reply:
Ahhh